src/repopage.c

```diff
@@ -704,11 +710,22 @@ repopagestore_load_page_range(Repopagestore *store, unsigned int pstart, unsigne
 #ifdef DEBUG_PAGING
 	  fprintf(stderr, "PAGEIN: %d to %d", pnum, i);
 #endif
+#ifndef _WIN32
           if (pread(store->pagefd, compressed ? buf : dest, in_len, store->file_offset + p->page_offset) != in_len)
 	    {
 	      perror("mapping pread");
 	      return 0;
 	    }
+#else
+	  DWORD read_len;
+	  OVERLAPPED ovlp;
+	  ovlp.Offset = store->file_offset + p->page_offset;
```

Comment:
Shouldn't you also zero `ovlp.OffsetHigh`?

Reply:
I added an initialization for OVERLAPPED, just like postgres does in pg_pread here: https://github.com/postgres/postgres/blob/74eb2176bf3ac7a2fae1282b4f4f1d000f3e6d78/src/port/pread.c#L29-L48